sdk/ai/ai-voicelive/samples/basic-web-voice-assistant/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "basic-web-voice-assistant",
+  "version": "1.0.0",
+  "description": "A simple web-based voice assistant demonstrating Azure Voice Live SDK real-time features",
+  "type": "module",
+  "main": "src/main.ts",
+  "scripts": {
+    "build": "tsc && vite build",
+    "dev": "vite",
+    "dev-sdk": "vite --force",
+    "preview": "vite preview",
+    "serve": "python -m http.server 8080",
+    "type-check": "tsc --noEmit",
+    "reload": "curl -X POST http://localhost:3000/__vite_ping 2>/dev/null || echo 'Vite server not responding'"
+  },
+  "dependencies": {
+    "@azure/ai-voicelive": "file:../../azure-ai-voicelive-1.0.0-beta.1.tgz",
```

Comment:
this probably needs to be replaced with a version specifier once we have our first release

Reply:
Yeah. Discussed.